.github/workflows/dependent-repos.yml

```diff
@@ -10,7 +10,7 @@ env:
 jobs:
   android-bindings:
     runs-on: [self-hosted, Linux, large]
-    container: mobilecoin/builder-install:v0.0.25
+    container: mobilecoin/builder-install:v0.0.26
```

Comment:
I guess the dependent repos don't yet build with 0.0.27?
If that's the case then whats causing it?

Reply:
nice catch!, that's a fail on my part when I bumped during development of this feature from 26 to 27

Reply:
updated now